.github/test-lib.yaml

```diff
@@ -0,0 +1,70 @@
+name: Run tests
+
+on:
+  push:
+    branches: [master]
+    paths:
+      - 'library/**'
+  pull_request:
+    branches: [master]
+    paths:
+      - 'library/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+
+    steps:
```

Comment:
maybe the working library for the needs to be changed for the commands

Reply:
I'm not sure what you mean?

Reply:
below you answered my concerns